assets/src/components/ladder.tsx

```diff
@@ -60,18 +51,69 @@ const Ladder = ({
       ? timepoints.slice().reverse()
       : timepoints
 
+  const timepointStatusY = timepointStatusYFromTimepoints(
+    timepoints,
+    timepointSpacingY
+  )
+  const { ladderVehicles, widthOfLanes } = ladderVehiclesFromVehicles(
+    vehicles,
+    ladderDirection,
+    timepointStatusY
+  )
+
+  const width = 120 + 2 * widthOfLanes * 1.2
```

Comment:
With the new more descriptive width calculations, this `* 1.2` scaling shouldn't be necessary, assuming we want constant sized margins.